plugins/module_utils/cohesity_hints.py

```diff
@@ -680,7 +683,7 @@ def unregister_source(module, self):
         headers = {
             "Accept": "application/json",
             "Authorization": "Bearer " + token,
-            "user-agent": "cohesity-ansible/v1.1.9",
```

Comment:
Can we define a constance somewhere to track the user agent value and refer to the constant value here?

Reply:
We can do that defeinitely, but currently trying to resolve several lint issues received in this build. This can be done separately as part of different pr maybe.